internal/lib/lib.go

```diff
@@ -651,7 +652,12 @@ func InformersToRegister(oclient *oshiftclient.Clientset, kclient *kubernetes.Cl
 		allInformers = append(allInformers, utils.NodeInformer)
 
 		informerTimeout := int64(120)
-		_, err := oclient.RouteV1().Routes("").List(context.TODO(), metav1.ListOptions{TimeoutSeconds: &informerTimeout})
+		_, err := kclient.CoreV1().Services("").List(context.TODO(), metav1.ListOptions{TimeoutSeconds: &informerTimeout})
+		if err != nil {
+			utils.AviLog.Warnf("can't access service objects, error : %s", err)
```

Comment:
We are anyway sending an error back to the caller and the caller prints the error message, so I am wondering whether we really need this log here?

Reply:
yeah, it's unnecessary, removed.